packages/web-components/fast-foundation/src/radio-group/radio-group.ts

```diff
@@ -120,6 +132,13 @@ export class RadioGroup extends FASTElement {
      */
     public connectedCallback(): void {
         super.connectedCallback();
+        this.setupRadioButtons();
+        this.parentToolbar = this.parentElement?.closest('[role="toolbar"]');
+        this.isInsideToolbar =
```

Comment:
What are the toolbar bits for?

Reply:
Radiogroup within toolbars have a slightly different interaction model per the W3C specification.